tools/halide_image_io.h

```diff
@@ -2088,31 +2090,32 @@ struct ImageTypeConversion {
         // as documentation and a backstop against breakage.
         static_assert(!ImageType::has_static_halide_type,
                       "This variant of convert_image() requires a dynamically-typed image");
+        constexpr int DimsUnconstrained = Internal::DimsUnconstrained;
```

Comment:
Would "AnyDims" be a better name than "DimsUnconstrained"?

Reply:
`Unconstrained` is the term we use in HalideBuffer.h so that's what I used here.

Reply:
I know, I was reconsidering the name in general. Seeing it in usage makes it seem clunky.

Reply:
Ah, gotcha. I think I originally called it "DynamicDims" and @zvookin suggested "DimsUnconstrained". "AnyDims" is fine with me too and definitely terser. I'll make the change everywhere if you like.